packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -296,6 +300,16 @@ class InteractiveViewer extends StatefulWidget {
   /// than maxScale.
   final double minScale;
 
+  /// Changes the scrolling behavior after end of a gesture.
+  ///
+  /// Used as the coefficient of friction in the inertial translation animation.
+  /// This value was eyeballed to give a feel similar to Google Photos.
```

Comment:
Can you remove this? This could change at any time, but also does not account for how Google Photos behaves on different devices.

Reply:
+1.  Keep it in the comments below, but remove it from the public docs.  You could make it "Google Photos on Android 12" since I think that's what I was using when I wrote that comment.